docs/Configuration.md

```diff
@@ -58,6 +58,7 @@ You can add these configurations into spark-defaults.conf to enable or disable t
 | spark.gluten.sql.columnar.fallback.ignoreRowToColumnar       | When true, the fallback policy ignores the RowToColumnar when counting fallback number.                                                                                                                                                                                                                                                                                                                                                                                                                                                   | true                                                 |
 | spark.gluten.sql.columnar.fallback.preferColumnar            | When true, the fallback policy prefers to use Gluten plan rather than vanilla Spark plan if the both of them contains ColumnarToRow and the vanilla Spark plan ColumnarToRow number is not smaller than Gluten plan.                                                                                                                                                                                                                                                                                                                      | true                                                 |
 | spark.gluten.sql.columnar.force.hashagg                      | Force to use hash agg to replace sort agg.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | true                                                 |
+| spark.gluten.sql.mergeTwoPhasesAggregate.enabled             | Whether to merge two phases aggregate if if there are no other operators between two aggregates.                                                                                                                                                                                                                                                                                                                                                                                                                                          | true                                                 |
```

Comment:
May be : `Whether to merge two phases aggregate if there are no other operators between two aggregates.` ,right ?

Reply:
addressed, thanks